app/lib/submission_builder/state1099_int.rb

```diff
@@ -14,10 +14,10 @@ def document
           end
         end
         xml.PayerEIN form1099int.payer_tin&.tr('-', '') if form1099int.payer_tin
-        xml.RecipientSSN form1099int.recipient_tin.tr('-', '') if form1099int.recipient_tin
-        recipient = if form1099int.recipient_tin.tr('-', '') == intake.primary.ssn
+        xml.RecipientSSN form1099int.recipient_tin&.tr('-', '') if form1099int.recipient_tin
+        recipient = if form1099int.recipient_tin&.tr('-', '') == intake.primary.ssn
```

Comment:
this line ' xml.RecipientSSN form1099int.recipient_tin&.tr('-', '') if form1099int.recipient_tin' is a little redundant so might be better to check for the existence of recipient_tin around the entire block